build.gradle

```diff
@@ -446,13 +446,16 @@ gradle.projectsEvaluated {
 }
 
 // test retry configuration
+boolean isCiServer = System.getenv().containsKey("CI")
 subprojects {
   apply plugin: "org.gradle.test-retry"
   tasks.withType(Test).configureEach {
     retry {
+      if (isCiServer) {
```

Comment:
BuildParams.isCI()

Reply:
Good call. Fixed here: #5368

I'll backport that one and close this PR.